### PR TITLE
format cart index page

### PR DESCRIPTION
**Pull request by Automatic365**

### CODE REVIEW

**Comment by dtinianow** [comment]
Very spacious page layout

**Comment by Automatic365** [comment]
I like to explore the studio space

**Comment by pwentz** [comment]
@Automatic365 does `col-xs...` make your dive a fraction of the size of the `col-md`?

File: app/views/cart/index.html.erb [hunk 1]
```diff
@@ -3,44 +3,63 @@
...
+				<div class="panel-heading">
+					<div class="panel-title">
+						<div class="row">
+							<div class="col-xs-6">
+								<h5><span class="glyphicon glyphicon-shopping-cart"></span> Shopping Cart</h5>
+							</div>
+							<div class="col-xs-6">
+								<a href="/" type="button" class="btn btn-primary btn-sm btn-block">
+									<span class="glyphicon glyphicon-share-alt"></span> Continue shopping
+								</a>
```
Comment (Yoyo2Code) [inline app/views/cart/index.html.erb]: Great choice of Glyphicon.

**Pull request merged**